LICENSE

```diff
@@ -1,16 +1,12 @@
 The following license applies to all source codes and assets, with the
 exception of Code.org's proprietary or licensed artwork, graphics, sounds,
 lectures, videos, logos, characters, and screenshots, or likenesses thereof
-("Proprietary Materials"), which are located in the following locations in
-this repository:
+("Proprietary Materials"). 
 
-  ./dashboard/app/assets/images
-  ./apps/static
-
-The Proprietary Materials may not be used or distributed or accessed in any
-manner other than directly via Code.org's own web site, mobile applications,
-or social media channels, as-is without modification or embedding in another
-format.
+The Proprietary Materials, a list of which [can be found here](NOTICE), may
+not be used or distributed or accessed in any manner other than directly via
+Code.org's own web site, mobile applications, or social media channels, as-is
+without modification or embedding in another format.
```

Comment:
3rd party partners know that their assets are part of our open-source repo, so I'm sure they understand that it will be used in developer environments. I'll update the list to include it explicitly.